examples/resources/env0_agent_project_assignment/resource.tf

```diff
@@ -0,0 +1,11 @@
+resource "env0_project" "example" {
```

Comment:
Note/Question:

Is there a way to test this in harness?
I don't believe I have a way to create agents. Open to suggestions.

At the moment there are no added integration tests.

Reply:
That wouldn't be ROI positive at the moment so I'd leave it without harness tests